### ISSUE TITLE
Brackets 2.01 on Windows 10 Does Not Remember Window Size & Position

### ISSUE THREAD

**Issue by ted-kal** [brackets-cont/brackets#123]
After upgrading the old Adobe version (Release 1.14.2), to v2.01, I have noticed that the application does not remember window size or position from your previous session when opening the program.

### Versions

I am running Windows 10 Pro v 20H2. I have installed Brackets v2.01 over the previous installation version of v1.14.2.

**Comment by charlypa** [comment]
@ted-kal 
Apologies for the late reply. Are you still facing the issue?

**Comment by ted-kal** [comment]
Yes I am still experiencing this issue. Thanks for your response.

**Comment by charlypa** [comment]
@ted-kal 
Apologies for the issue.
Are you facing this issue after the upgrade every time you close and open the app?

**Comment by ted-kal** [comment]
Yes, it happens every time I close and open the app.
